core/jobs/batch_jobs/suggestion_stats_computation_jobs.py

```diff
@@ -269,6 +270,564 @@ def _generate_translation_contribution_model(
             return translation_contributions_stats_model
 
 
+class GenerateTranslationContributionStatsJob(base_jobs.JobBase):
```

Comment:
Please add at least some tests, you will find out whether this works or not.

Reply:
Done